go.mod

```diff
@@ -1,26 +1,44 @@
 module github.com/aserto-dev/service-host
 
+// replace github.com/aserto-dev/aserto-grpc => ../aserto-grpc
+
 go 1.19
 
 require (
+	github.com/aserto-dev/aserto-grpc v0.2.2
 	github.com/aserto-dev/certs v0.0.2
 	github.com/aserto-dev/go-http-metrics v0.10.1-20221024-1
-	github.com/grpc-ecosystem/grpc-gateway/v2 v2.7.3
+	github.com/grpc-ecosystem/go-grpc-middleware/providers/prometheus v1.0.0-rc.0
```

Comment:
Why are we using rc0 if v1.4.0 is available?

Reply:
v1.0.0-rc.0 is latest release for prometheus provider